ddtrace/contrib/cherrypy/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+The Cherrypy trace middleware will track request timings.
+It uses the cherrypy hooks and creates a tool to track requests and errors
+
+To install the middleware, add::
+
+    from ddtrace import tracer
+    from ddtrace.contrib.cherrypy import TraceMiddleware
+
+and create a `TraceMiddleware` object::
+
+    traced_app = TraceMiddleware(cherrypy, tracer, service="my-cherrypy-app")
+
+Here is the end result, in a sample app::
+
+    import cherrypy
+
+    from ddtrace import tracer, Pin
+    from ddtrace.contrib.cherrypy import TraceMiddleware
+    TraceMiddleware(cherrypy, tracer, service="my-cherrypy-app")
+
+    @cherrypy.tools.tracer()
+    class HelloWorld(object):
+        def index(self):
+            return "Hello World"
+        index.exposed = True
+
+    cherrypy.quickstart(HelloWorld())
```

Comment:
Can we also add a configuration section with the options we have? Starlette is a good example https://github.com/DataDog/dd-trace-py/blob/bd60bb397b624e71ee0ac9797cf473e640336637/ddtrace/contrib/starlette/__init__.py#L1-L75

Reply:
Added!